src/gui/EQ/EQPlot.cpp

```diff
@@ -240,6 +280,12 @@ void EQPlot::setSelectedBand (int bandIndex)
 
 void EQPlot::paint (juce::Graphics& g)
 {
+    const float maxGain = 18.0f;
```

Comment:
I think we can get this from `params.maxMagnitudeDB`?

Reply:
I think `params.maxMagnitudeDB` is currently 30dB? I couldn't find one that specifies 18dB as the max, but let me know if there's something I've missed

Reply:
Oh good call, it looks like we're setting it to 20 dB [here](https://github.com/Chowdhury-DSP/ChowMultiTool/blob/main/src/gui/EQ/EQPlot.cpp#L72). So maybe the answer is to make this `static constexpr auto labelGain = 18.0f;` (or something like that, there's probably a better name out there somewhere)

Reply:
I'd go for `labelGainDb`,  `labelMagnitudeDb` or `gainIndicatorDb`, vote now!

Reply:
how about I change `drawDbLabels` to `drawMagnitudeLabels` to be consistent with `drawMagnitudeLines` and call the max gain variable `labelMagnitudeDb`?

Reply:
Yeah, that seems like a good solution!